client/src/app/site/motions/motion-list/motion-list.component.ts

```diff
@@ -121,6 +122,15 @@ export class MotionListComponent extends BaseComponent implements OnInit {
         }
     }
 
+    isDisplayIcon(state): boolean {
+        const stateName = state.name;
+        if (stateName === 'accepted' || stateName === 'rejected' || stateName === 'not decided') {
```

Comment:
Just a `return (stateName === 'accepted' || stateName === 'rejected' || stateName === 'not decided');` is enough.